app/assets/stylesheets/components/_navbar.scss

```diff
@@ -4,19 +4,23 @@ nav {
   justify-content: space-evenly;
   // position: relative;
   width: 95%;
-  margin: 0 auto;
-  margin-top: 10px;
+  margin: 10px auto 0;
   height: 70px;
   padding: 16px;
   background-color: rgb(232, 230, 230);
   border-radius: 15px;
+  .logo{
+    height: 40px;
+    padding-left: 10px;
 
+  }
   a{
     font-family: 'roboto', sans-serif;
```

Comment:
Do you really need to set the font-family when it has already been set on the body in the _setup.scss file ?
Try without it and see if it changes.
If it is required please use `font-family: inherit;` instead